doc/install/mirrors.rst

```diff
@@ -15,7 +15,6 @@ Locations
 These mirrors are available on the following locations:
 
 - **EU: Netherlands**: http://eu.ceph.com/
-- **AU: Australia**: http://au.ceph.com/
```

Comment:
@wido could you help confirm this? is this transient issue ? DNS still resolves, but seems the web server is not accessible anymore.